science/ngspice/Portfile

```diff
@@ -19,38 +18,48 @@ long_description \
 homepage        http://ngspice.sourceforge.net/
 master_sites    sourceforge:project/ngspice/ng-spice-rework/${version}
 
-checksums       rmd160  094d76e789b65d10e7d80f2f2f90397ed0ec13c9 \
-                sha256  4f818287efba245341046635b757ae81f879549b326a4316b5f6e697aa517f8c \
-                size    8450071
+# generated via openssl dgst -rmd160 ngspice-43.tar.gz 
+
```

Comment:
No need for such a comment.
```suggestion
# generated via openssl dgst -rmd160 ngspice-43.tar.gz 

```
No need to use `openssl` to generate checksums either. MacPorts will do it for you if you run `sudo port -d checksum`. It will even format it for easy pasting into the Portfile.

Reply:
I see.  remove the line.